lib/matplotlib/widgets.py

```diff
@@ -1061,6 +1061,8 @@ def disconnect(self):
 
     def clear(self, event):
         """clear the cursor"""
+        if self.ignore(event):
```

Comment:
If this behavior is consistent with the other widgets then disregard my last comment.